src/selector_manager/model/Selector.ts

```diff
@@ -191,7 +191,7 @@ export default class Selector extends Model<SelectorProps & { [key: string]: unk
    * @private
    */
   static escapeName(name: string) {
-    return `${name}`.trim().replace(/([^a-z0-9\w-\\:@\\/]+)/gi, '-');
+    return `${name}`.trim().replace(/([^a-z0-9\w\-\\:@\\/()\.%\+\[\]]+)/gi, '-');
```

Comment:
Can you add appropriate tests here `test/specs/selector_manager/model/SelectorModels.ts`, please?

Reply:
Yes. I'll add those.